.gitpod.yml

```diff
@@ -0,0 +1,16 @@
+# Commands to start on workspace startup
+tasks:
+  - name: Preparing Environment
+    init: | 
+      make php
+      make
+      go install
```

Comment:
Since this is for development, why do we need to make the binary and install it?

Alternatively, you could `make single` which also downloads the correct `.phar`.

Reply:
I'll defer to your expertise on this. I lack meaningful experience in Go, so I consider this MR more of a Gitpod template for enabling easier contribution.

Any revision by you is welcome.

Reply:
If I recall correctly, make was used to ensure the appropriate files were in place to enable the CLI to run in Gitpod